### PR TITLE
Master fix seb

### PR DESCRIPTION
**Pull request by seb-odoo**
rebase-ff

### CODE REVIEW

**Comment by seb-odoo** [comment]
@qsm-odoo very small PR with some fixes from my image task, better to merge those separately to ease the review of the other one

Maybe some of those need to be fixed in earlier versions?

File: addons/website/static/src/js/content/snippets.animation.js [hunk 1]
```diff
@@ -860,6 +860,10 @@ registry.gallerySlider = publicWidget.Widget.extend({
     destroy: function () {
         this._super.apply(this, arguments);
 
+        if (!this.$indicator) {
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/snippets.animation.js]: Probably for << master ?

File: addons/web_editor/static/src/js/wysiwyg/widgets/media.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ odoo.define('wysiwyg.widgets.media', function (require) {
 var core = require('web.core');
 var Dialog = require('web.Dialog');
 var fonts = require('wysiwyg.fonts');
+var session = require('web.session');
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/js/wysiwyg/widgets/media.js]: Merging fja's PR in saas-12.2 which fixes this: https://github.com/odoo/odoo/pull/33247

File: addons/web_unsplash/static/src/js/unsplash_image_widget.js [hunk 3]
```diff
@@ -300,6 +300,7 @@ widgetsMedia.ImageWidget.include({
             }
         }
         this._super.apply(this, arguments);
+        this.hasSearched = true;
```
Comment (qsm-odoo) [inline addons/web_unsplash/static/src/js/unsplash_image_widget.js]: Well it is changed... https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/wysiwyg/widgets/media.js#L73

File: odoo/tools/image.py [hunk 4]
```diff
@@ -243,7 +243,7 @@ def image_process(base64_source, size=(0, 0), verify_resolution=False, quality=8
             image.resize(max_width=size[0], max_height=size[1])
     if colorize:
         image.colorize()
-    return image.image_base64(quality=quality, output_format=output_format)
+    return image.image_base64(quality=min(quality, 95), output_format=output_format)
```
Comment (qsm-odoo) [inline odoo/tools/image.py]: So 97 is really not better than 95 ? If it is really the case, I think it is worth a comment in the code

File: odoo/tools/image.py [hunk 5]
```diff
@@ -288,7 +288,7 @@ def is_image_size_above(base64_source, size=IMAGE_BIG_SIZE):
     """
     if not base64_source:
         return False
-    if base64_source[:1] == b'P':
+    if base64_source[:1] == b'P' or base64_source[:1] == 'P':
```
Comment (qsm-odoo) [inline odoo/tools/image.py]: `in (b'P', 'P')` ? Don't know.

Comment (seb-odoo) [reply odoo/tools/image.py]: Ok for using `in` but we decided to fix this in 12.0 instead too.

File: addons/web_editor/static/src/js/wysiwyg/widgets/media.js (hunk 2 above)
Comment (seb-odoo) [reply addons/web_editor/static/src/js/wysiwyg/widgets/media.js]: Ok I'll wait forward port.

File: odoo/tools/image.py (hunk 4 above)
Comment (seb-odoo) [reply odoo/tools/image.py]: Indeed, see `quality` in https://pillow.readthedocs.io/en/4.0.x/handbook/image-file-formats.html#jpeg

I'll add a comment

**Comment by seb-odoo** [comment]
Well I'm closing this since most of those will be forward-ported, the few that are remaining can stay on my main PR.

Thanks for the input.

**Pull request closed without merging**